apps/css-workshop/package.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {},
   "devDependencies": {
     "@itwin/itwinui-css": "*",
-    "@itwin/itwinui-icons-elements": "0.2.0",
+    "@itwin/itwinui-icons-elements": "0.20.0",
```

Comment:
this will likely cause visual tests to fail because of some icon changes. best done in a separate PR

Reply:
#1732 is now merged